lib/ecto_job/worker.ex

```diff
@@ -17,12 +18,18 @@ defmodule EctoJob.Worker do
   Start a worker process given a repo module and a job struct
   This may fail if the job reservation has expired, in which case the job will be
   reactivated by the producer.
+  SimpleDemo.JobQueue.enqueue("jim")
   """
   @spec start_link(repo, EctoJob.JobQueue.job(), DateTime.t()) :: {:ok, pid}
   def start_link(repo, job = %queue{}, now) do
     Task.start_link(fn ->
       with {:ok, job} <- JobQueue.update_job_in_progress(repo, job, now) do
-        queue.perform(JobQueue.initial_multi(job), job.params)
+        case queue.perform(JobQueue.initial_multi(job), job.params) do
```

Comment:
How about handling exceptions thrown from `queue.perform`?

Reply:
Good idea, I would like to implement,  but I would need more know how of the try/rescue feature.